site/src/sphinx/server-access-log.rst

```diff
@@ -0,0 +1,145 @@
+.. _ServerBuilder: apidocs/index.html?com/linecorp/armeria/server/ServerBuilder.html
+.. _`RequestLog`: apidocs/index.html?com/linecorp/armeria/common/logging/RequestLog.html
+
+.. _server-access-log:
+
+Writing an access log
+=====================
+
+Configuring logging framework
+-----------------------------
+
+To write an access log, you need to configure a logging framework first. The following configurations are
+simple examples of ``logback`` and ``log4j`` respectively.
+
+logback
+^^^^^^^
+
+.. code-block:: xml
+
+    <configuration>
+      <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+          <pattern>%msg%n</pattern>
+        </encoder>
+      </appender>
+
+      <logger name="com.linecorp.armeria.logger.access" level="INFO" additivity="false">
+        <appender-ref ref="CONSOLE" />
+      </logger>
+
+      <root level="debug">
+        <appender-ref ref="STDOUT" />
+      </root>
+    </configuration>
+
+
+log4j
+^^^^^
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Configuration status="WARN">
+      <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+          <PatternLayout pattern="%msg%n"/>
+        </Console>
+      </Appenders>
+      <Loggers>
+        <Logger name="com.linecorp.armeria.logger.access" level="info" additivity="false">
+          <AppenderRef ref="Console"/>
+        </Logger>
+        <Root level="debug">
+          <AppenderRef ref="Console"/>
```

Comment:
- Shouldn't we use different appender with better pattern layout?
- How about using a file-based appender for access logs?